dbms/src/Interpreters/evaluateMissingDefaults.cpp

```diff
@@ -59,4 +66,24 @@ void evaluateMissingDefaults(Block & block,
     }
 }
 
+
+void evaluateMissingDefaultsUnsafe(Block & block,
```

Comment:
Why it's unsafe?

Reply:
It doesn't save block before changes. The original function makes copy of block to prevent it's mutations in expression analyser.